### PR TITLE
Weather-App

### PR DESCRIPTION
**Pull request by Temmarie**
In this project, I built a weather app using the OpenWeatherMap API.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+function setQuery(evt) {
+  if (evt.keyCode === 13) {
+    getResults(searchbox.value);//eslint-disable-line
+  }
+}
+
+function getResults(query) {
+  fetch(`${api.base}weather?q=${query}&units=metric&APPID=${api.key}`)
+    .then(weather => weather.json()).then(displayResults);//eslint-disable-line
+}
```
Comment (trekab) [inline src/main.js]: Make use of ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)). You could use arrow functions in this case.

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @Temmarie! Your project looks good and much as you were able to implement most of the functionality, there are some key requirements that are missing in your project. These include:
- [x] Add a toggle control to determine if the result will be shown in Celsius or Fahrenheit  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [x] Make use either `promises` or `async/await`. A core part of this project is learning how to work with asynchronous communication in JS. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/promises_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [x] Add necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
- [x] Make use of `webpack` to build your project. Change the way you use the `main.js` file in the `index.html` file. You should have an entry point JS file in the dist folder whose code is generated from your `src` JS files by `webpack` usually by using the `npm run build` command.
- [x] Make use of ES6 syntax as indicated in the comment below.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: src/main.js (hunk 1 above)
Comment (Temmarie) [reply src/main.js]: Thanks for the review. I've made all of the required changes.

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Temmarie 
### You have done well so far :clap: However, you did not finish all the previous review changes. Kindly work on them with the new ones and submit them for another code review.
- [x] Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
- [x] It has a toggle control to determine if the result will be shown in Celsius or Fahrenheit
- Happy coding!

**Comment by Temmarie** [comment]
Thanks for the review, I have made all the necessary changes, as requested by the previous TSE, which includes: 
- Adding necessary run common commands like build, development in package.json inside `scripts'.
- Adding the temperature toggle which works by clicking on the current temperature.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Temmarie :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### Please don't forget to build your project and update the Live Link!

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**